src/screens/DiscoveryScreenV2/elements/DiscoveryTitleSeparator.js

```diff
@@ -47,6 +48,8 @@ const DiscoveryTitleSeparator = (props) => {
   return (
     <View
       style={{
+        borderBottomColor: withBorderBottom && !rotateArrow ? COLORS.gray210 : 'transparent',
+        borderBottomWidth: 1,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The conditional application of the borderBottomColor based on the `withBorderBottom` and `rotateArrow` props is a good approach. However, consider extracting this logic into a variable for better readability.
```diff
-        borderBottomColor: withBorderBottom && !rotateArrow ? COLORS.gray210 : 'transparent',
+        const borderBottomColorValue = withBorderBottom && !rotateArrow ? COLORS.gray210 : 'transparent';
+        borderBottomColor: borderBottomColorValue,
         borderBottomWidth: 1,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->